docs/source/ProgrammingGuide/View.rst

```diff
@@ -458,6 +460,15 @@ in a way that reflects how first-touch affinity to threads is desired.
 Typically it is sufficient to use the parallel iteration index as the index of the location in the
 view to write to.
 
+.. warning::
+
+  :cppkokkos:`WithoutInitialization` implies that the destructor of each element of the :cppkokkos:`View` **will not be called**.
+  For instance, if you need the destructor of your :code:`MyObjectWithDestructor` be called for each element of your :cppkokkos:`View<MyObjectWithDestructor*>`,
+  you **should not use** :cppkokkos:`WithoutInitialization`.
+
+  The reason for that is, since `Kokkos` did not initialize the memory, it cannot know onto which elements of the :cppkokkos:`View` calling the destructor
+  of the :code:`value_type` is valid.
```

Comment:
Maybe, rather something like
```suggestion
The mental model is that whenever placement new is used to call the constructor, the destructor also isn't called before the memory is deallocated but it needs to be be called manually.
```
Or omit it with my previous suggestion.

Reply:
:heavy_check_mark: